dasgo/events.go

```diff
@@ -394,66 +398,68 @@ type MessageUpdate struct {
 // Message Delete
 // https://discord.com/developers/docs/topics/gateway#message-delete
 type MessageDelete struct {
-	MessageID Snowflake `json:"id,omitempty"`
-	ChannelID Snowflake `json:"channel_id,omitempty"`
+	MessageID Snowflake `json:"id"`
+	ChannelID Snowflake `json:"channel_id"`
 	GuildID   Snowflake `json:"guild_id,omitempty"`
 }
 
 // Message Delete Bulk
 // https://discord.com/developers/docs/topics/gateway#message-delete-bulk
 type MessageDeleteBulk struct {
-	MessageIDs []Snowflake `json:"ids,omitempty"`
-	ChannelID  Snowflake   `json:"channel_id,omitempty"`
+	MessageIDs []Snowflake `json:"ids"`
+	ChannelID  Snowflake   `json:"channel_id"`
 	GuildID    Snowflake   `json:"guild_id,omitempty"`
 }
 
 // Message Reaction Add
 // https://discord.com/developers/docs/topics/gateway#message-reaction-add
 type MessageReactionAdd struct {
-	UserID    Snowflake    `json:"user_id,omitempty"`
-	ChannelID Snowflake    `json:"channel_id,omitempty"`
-	MessageID Snowflake    `json:"message_id,omitempty"`
+	UserID    Snowflake    `json:"user_id"`
+	ChannelID Snowflake    `json:"channel_id"`
+	MessageID Snowflake    `json:"message_id"`
 	GuildID   Snowflake    `json:"guild_id,omitempty"`
 	Member    *GuildMember `json:"member,omitempty"`
-	Emoji     *Emoji       `json:"emoji,omitempty"`
+	Emoji     *Emoji       `json:"emoji"`
 }
 
+// TODO: member not included in chart so I copied how it was treated above
```

Comment:
https://discord.com/developers/docs/topics/gateway#message-reaction-remove-message-reaction-remove-event-fields has no member field so it should be removed from `MessageReactionRemove` struct.